docs/markdown/ios/card/subcomponents/card.pins.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Card.Pins
+description: The Boards preview block or “Boards PB” utilizes two of the Board and Section components to present a collection of imagery that typically represents a Card.Boards.
+fullwidth: true
+---
+
+<ImgContainer src="https://github.com/pinterest/gestalt/assets/96082362/ec0692aa-cc1a-42df-80c9-f0e212100fe4" alt="Example of a Card container with a Card.Boards preview that contains two Card.Boards representations." noPadding color="background-elevation-accent"/>
+
+
+## Usage guidelines
+
+<TwoCol>
+<Group>
+<Do title="When to use"/>
+- To present multiple entry points into a single destination containing Pins (ex: stream)
+- When the user goal is best supported by focusing them on one Pin at a time (ex: watching videos in a related stream, evaluating Pins with a high intent mindset)
+
+</Group>
+<Group>
+<Dont title="When not to use" />
+- When each tap target leads to a different destination (ex: different streams)
+- When the goal is to direct Pinners to a landing experience that displays more than one Pin (e.g. a topical Pin feed). 
+</Group>
+</TwoCol>
+
+## Best practices
+<TwoCol>
```

Comment:
It's a little weird to have a "Do" slotted underneath a "Don't"
![Screen Shot 2023-06-14 at 4 27 56 PM](https://github.com/pinterest/gestalt/assets/12059539/d1f9d109-d816-469b-832b-21c570cfcef3)

Reply:
ugh. yes. I asked about this one. They didn't have matching do's/dont's like we should. let me ask the designers. Since there are no images, can all of these just be bullets.